hloc/visualization.py

```diff
@@ -72,15 +72,16 @@ def visualize_loc(results, image_dir, reconstruction=None,
 
     for qname in selected:
         loc = logs['loc'][qname]
-        visualize_loc_from_log(image_dir, qname, loc, reconstruction, **kwargs)
+        visualize_loc_from_log(image_dir, db_image_dir, qname, loc, reconstruction, **kwargs)
```

Comment:
This should be `visualize_loc_from_log(image_dir, qname, loc, reconstruction,  db_image_dir, **kwargs)` I think.

Reply:
oops! yes